content/documentation/device-examples/LEDMatrix.md

```diff
@@ -0,0 +1,52 @@
+---
+title: LEDMatrix
+weight: 207
+tags: ["LEDMatrix"]
+---
+### Description
+The [LEDMatrix](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/example/components) (src/main/java/com/pi4j/example/components) is a template class, that you can use in your own Java-project.
+It is an extension of the class [LEDStrip](/documentation/device-examples/ledstrip). You can use many Neopixel-LED-Strips, cut them in between the LEDs, and sold them back together with wires.
+Like this, you can generate a matrix on your own, with many strips wired together.
+
+{{% notice note %}}
+Make sure to check if SPI is enabled in your RaspberryPI.
+Check the SPI Address. Default is "SPI0 MOSI" Pin (#19).
+{{% /notice %}}
+
+### Layout
+![LEDStrip Layout](/assets/documentation/device-examples/Layout-LEDStrip.png)
```

Comment:
layout is not matching with code. show all three strips and connect them with each other. Use Resistor to protect mosi output